mackup/applications/intellijidea.cfg

```diff
@@ -15,10 +15,21 @@ Library/Preferences/IntelliJIdea14
 Library/Application Support/IntelliJIdea15
 Library/Preferences/IntelliJIdea15
 Library/Application Support/IntelliJIdea2016.1
+Library/Preferences/IdeaIC2016.1
+Library/Preferences/IdeaIC2016.2
+Library/Preferences/IdeaIC2016.3
 Library/Preferences/IntelliJIdea2016.1
 Library/Application Support/IdeaIC2016.1
-Library/Preferences/IdeaIC2016.1
 Library/Preferences/IntelliJIdea2016.2
 Library/Application Support/IntelliJIdea2016.2
 Library/Preferences/IntelliJIdea2016.3
 Library/Application Support/IntelliJIdea2016.3
+Library/Preferences/IdeaIC2017.1
+Library/Preferences/IdeaIC2017.2
+Library/Preferences/IdeaIC2017.3
+Library/Preferences/IntelliJIdea2017.1
+Library/Application Support/IdeaIC2017.1
```

Comment:
Are you running community edition? Because I am, and am showing "Idea2017.1" in App Support.